elf.h

```diff
@@ -17,44 +17,45 @@
 
 #define PT_LOAD 0x00000001u
 
+/* Note, only little endian ELFs handled */
```

Comment:
Would it make sense for `elf2uf2` and `picotool` to both pick up a common `elf.h` header from `pico-sdk`? :shrug: 
(These PRs are a bit over my head, so apologies if that's a silly suggestion.)